### DISCUSSION TITLE
Submission of issues does not work.

### DISCUSSION THREAD

**Discussion by mmoser18** [Devolutions/UniGetUI#3005] [General]
I tried to submit an issue (UniGetUI just tried to install me an ARM-version of an application on an Intel x64 box) but despite of having entered all required information the "Submit issue" button never became active. Why can't one submit issues on this repo? :-(

**Comment by marticliment** [comment]
Have you checked all the required checkboxes at the top and filled all the required text boxes?

**Comment by mmoser18** [comment]
Gush - I had overlooked those (or rather I had checked all but one of them). ||-( Sorry for the bandwidth consumed.
